docs/mutators.md

```diff
@@ -1,12 +1,14 @@
 ---
```

Comment:
Renaming this file will break the website. I'd suggest keeping this mutations for now as these docs contain both the mutators and the specific mutations

Reply:
Alright. I removed all the renamings, only left the _custom_edit_url_ change in that regard, I guess it should fix this issue here:
![image](https://user-images.githubusercontent.com/5451366/140080753-123662ce-86cb-4b2e-be71-bcffad0ba45d.png)